lib/core/base/context.js

```diff
@@ -201,9 +203,11 @@ function validateContext(context) {
  * @param {Object} spec Configuration or "specification" object
  */
 function Context(spec) {
-	/* eslint max-statements:["error",19], no-unused-vars:0 */
+	/* eslint max-statements:["error",22], no-unused-vars:0 */
 	'use strict';
-	var self = this;
+	if (!axe._tree) {
```

Comment:
Why do you need this here? I'd rather add it to the tests that are failing directly than have potential side effects that are difficult to track down.